contracts/utils/math/Math.sol

```diff
@@ -58,8 +60,11 @@ library Math {
      */
     function tryDiv(uint256 a, uint256 b) internal pure returns (bool success, uint256 result) {
         unchecked {
-            if (b == 0) return (false, 0);
-            return (true, a / b);
+            success = b > 0;
+            assembly ("memory-safe") {
+                // In EVM any value divided by zero is zero.
```

Comment:
This is exclusive to assembly afaiu, not the EVM.

```suggestion
                // In assembly any value divided by zero is zero.
```

Reply:
Alternatively

```suggestion
                // In yul any value divided by zero is zero.
```

Reply:
yul is a solidity specify thing. I'd argue this is about the opcode behavior.

Reply:
```suggestion
                // The `DIV` opcode returns zero when the denominator is 0.
```